crates/polars-sql/src/functions.rs

```diff
@@ -173,6 +173,10 @@ pub(crate) enum PolarsSqlFunctions {
     /// ```
     Round,
 
+    // Date Functions
+    // ----
+    Date,
```

Comment:
can you add the related sql expression to the docstring like the other variants?

Reply:
Oops - missed that, thank you.